### DISCUSSION TITLE
使用原有的DB模型进行加载训练

### DISCUSSION THREAD

**Discussion by lyc728** [PaddlePaddle/PaddleOCR#12179] [Q&A]
你好，我使用了ch_det_res18_db_v2.0进行训练，也下载了相应的模型，训练大概200轮和600轮进行预测时发现很多空白处误检
![5c220153-a615-4379-9f41-bb9f86493ef8](https://user-images.githubusercontent.com/88084847/137658416-f0b4b8be-cb94-4596-b50c-3e61caa44110.jpg)

**Comment by LDOUBLEV** [comment]
目前这个状态的精度有多少，

可以调整后处理参数，稍微加大一点后处理阈值，https://github.com/PaddlePaddle/PaddleOCR/blob/507129207b854b333d575839b8ce4cfa296a1411/configs/det/ch_ppocr_v2.0/ch_det_res18_db_v2.0.yml?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L55

**Comment by lyc728** [comment]
> 目前这个状态的精度有多少，
> 
> 可以调整后处理参数，稍微加大一点后处理阈值，https://github.com/PaddlePaddle/PaddleOCR/blob/507129207b854b333d575839b8ce4cfa296a1411/configs/det/ch_ppocr_v2.0/ch_det_res18_db_v2.0.yml?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L55

上次精度大概是0.8左右，没有记录下来，也尝试了加大后处理，发现效果也不明细，我是在街景数据下有个训练好的模型下进行微调的

**Comment by lyc728** [comment]
thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5
你好，参数我改成  thresh: 0.5
  box_thresh: 0.75
  max_candidates: 1000
  unclip_ratio: 1.5
box的误检少了，但是少了些框的检测，同时我还想问下，shrink_ratio: 0.4这里设置是否有影响呢？

**Comment by LDOUBLEV** [comment]
shrink_ratio: 0.4 训练的时候会影响；
box_thresh: 0.75 这个值减小一些试试，这个值太大会导致一些框被过滤掉

**Comment by LDOUBLEV** [comment]
每训练几个epoch，可以在测试集后上评估你的精度，保存精度最好的模型

**Comment by lyc728** [comment]
> ink_ratio: 0.4 训练的时候

因为误检情况还是存在，后处理部分好像改了并没有作用，是不是模型本身的问题

**Comment by lyc728** [comment]
> 每训练几个epoch，可以在测试集后上评估你的精度，保存精度最好的模型

精度最好模型  还是存在上面错误的情况
